src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -332,14 +352,13 @@ private String createTooltipTextForProject(final Project p) {
    private GridPane setUpEditProjectGridPane(final Project p, final Dialog<Project> dialog) {
       GridPane grid;
       final FXMLLoader loader = new FXMLLoader(Resources.getResource(RESOURCE.FXML_MANAGE_PROJECT));
+      loader.setControllerFactory(model.getSpringContext()::getBean);
```

Comment:
we should extract the duplicate loader functionality some time into another class